apps/condo/domains/common/schema/fields.js

```diff
@@ -11,11 +12,22 @@ const DV_FIELD = {
 const SENDER_FIELD = {
     type: Json,
     schemaDoc: 'Client-side devise identification used for the anti-fraud detection. ' +
-        'Example `{ dv: \'1\', fingerprint: \'VaxSw2aXZa\'}`. ' +
+        'Example `{ dv: 1, fingerprint: \'VaxSw2aXZa\'}`. ' +
         'Where the `fingerprint` should be the same for the same devices and it\'s not linked to the user ID. ' +
         'It\'s the device ID like browser / mobile application / remote system',
     isRequired: true,
     kmigratorOptions: { null: false },
+    hooks: {
+        validateInput: (args) => {
+            if (!hasValidJsonStructure(args, true, 1, {
+                fingerprint: {
```

Comment:
Missing validation on presence of `dv` field.

Reply:
it's inside `hasValidJsonStructure` logic %)

Reply:
ok ;)